cmd/collectors/rest/plugins/snapmirror/snapmirror.go

```diff
@@ -47,28 +51,60 @@ func (my *SnapMirror) Init() error {
 
 	my.query = "api/private/cli/volume"
 	my.svmVolToNode = make(map[string]string)
+	my.svmPeerDataMap = make(map[string]string)
+
+	my.data = matrix.New(my.Parent+".SnapMirror", "snapmirror", "snapmirror")
+
+	exportOptions := node.NewS("export_options")
+	instanceLabels := exportOptions.NewChildS("instance_labels", "")
+	instanceKeys := exportOptions.NewChildS("instance_keys", "")
+
+	if exportOption := my.ParentParams.GetChildS("export_options"); exportOption != nil {
```

Comment:
double check if `Merge` or `Union` could be used to merge these

Reply:
Yes, it could help when we need to combine both of the data. But here we need labels to instanceLabels and keys to instanceKeys.